addon/pods/components/frost-notification/template.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="notification-wrapper">
+	<div class="content">
```

Comment:
If you started from a `frost-` component, it's quite possible the TABs were already there.